src/main/resources/public/partials/admin_employees.html

```diff
@@ -188,35 +186,35 @@ <h2 class="view-title text-left">{{ 'employees_view.title' | translate}}</h2>
                             </div>
                         </div>
 
-                        <!--User history-->
+                        <!--User historyLog-->
```

Comment:
im not sure if you wanted to refactor this (and the variables below)

Reply:
yes and no - this line no, but below yes